### PR TITLE
Add Delegate Satement Creators w/ zero  VP to delegates page

### PR DESCRIPTION
**Pull request by jefag**

### CODE REVIEW

**Comment by jefag** [comment]
@stepandel, @andreitr , this patch [db7458e](https://github.com/voteagora/agora-next/pull/420/commits/db7458ed19bbec842f202625df34f428aa0b5036) is the heart of it.  everything else is re-factoring so that I could iterate faster.

**Review by andreitr — COMMENTED** [review]
Looks good. I am going to do more QA tomorrow but at the first glance LGTM. I also noticed duplicate results but that's probably due to the fact that you haven't merged the latest.

File: src/app/api/common/delegates/getDelegates.ts [hunk 1]
```diff
@@ -95,149 +95,157 @@ async function getDelegates({
       )`
       : "";
 
+  const includeZeroVPDelegates = true;
+
+  let delegateUniverseCTE: string;
+
+  const tokenAddress = contracts.token.address;
+
+  if (includeZeroVPDelegates) {
```
Comment (stepandel) [inline src/app/api/common/delegates/getDelegates.ts:104]: do we need this condition? seems like it's hardcoded to `true`. Also I don't really see a use case where we wouldn't include it

**Review by stepandel — APPROVED** [review]
I like this approach -- functionality looks good. just a couple of points to keep the code clean
could you also remove console.log s please

File: src/app/api/common/delegates/getDelegates.ts [hunk 2]
```diff
@@ -95,149 +95,157 @@ async function getDelegates({
       )`
       : "";
 
+  const includeZeroVPDelegates = true;
+
+  let delegateUniverseCTE: string;
+
+  const tokenAddress = contracts.token.address;
```
[the remark this answers is not in the document]
Comment (andreitr) [reply src/app/api/common/delegates/getDelegates.ts:102]: @stepandel, TenantContract  class converts the address to lower case in the getter 

```
  get address(): string {
    return this._address.toLowerCase();
  }
```

File: src/app/api/common/delegates/getDelegates.ts (hunk 1 above)
Comment (jefag) [reply src/app/api/common/delegates/getDelegates.ts:104]: It was only there as a backup.  to show the logic to undo this, if performance was terrible for one tenant.

Comment (jefag) [reply src/app/api/common/delegates/getDelegates.ts:104]: I'll back it out in the rebase.  This thread is going to get hidden when I do so.

**Pull request merged**